d2l-input-date-style-modules.html

```diff
@@ -88,6 +88,7 @@
 		<style include="vaadin-month-calendar-default-theme">
 			:host {
 				font-family: inherit;
+				font-size: 14px;
```

Comment:
Fixes an issue with the calendar days sizes overlapping/colliding if the size is inherited from d2l-typography.  The other calendar sizes besides these ones are already a fixed size.